utils/helm/speckle-server/templates/server/deployment.yml

```diff
@@ -276,6 +276,25 @@ spec:
           - name: EMAIL_FROM
             value: "{{ .Values.server.email.from }}"
           {{- end }}
+          
+          # *** Mailchimp ***
```

Comment:
cc @iainsproat: this is where your magic will be needed. PS: we have these values in the shared vault.